packages/orbit-components/src/AirportIllustration/__tests__/index.test.tsx

```diff
@@ -24,21 +24,21 @@ describe(`AirportIllustration of ${name}`, () => {
   });
 
   it("should have passed props", () => {
-    expect(screen.getByRole("img")).toHaveAttribute("alt", alt);
-    expect(screen.getByRole("img")).toHaveAttribute("data-test", dataTest);
+    expect(screen.getByRole("presentation")).toHaveAttribute("alt", alt);
```

Comment:
It does not make sense for an image to have role presentation and alt attribute. If anything, please keep the selectors and add `role="img"` on the test

Reply:
> If anything, please keep the selectors and add role="img" on the test

That was my original idea, but in this case I would need to add a `role` prop to AirportIllustration. Do we want to add this new prop to AirportIllustration? Last week when I was working on AirportIllustration I got impression that we want to keep `role` attribute out of this component.

Or I can rewrite this target from:
```
screen.getByRole("img")
```
to:
```
screen.getByTestId(dataTest)
```

I think second option could be better for our use case. WDYT?
@DSil

Reply:
Yes, second option for sure

Reply:
Actually… I just noticed your question and it presents a problem!

If we have **IllustrationPrimitive** with `role="presentation"` by default and if we can't define the `role` for **AirportIllustration**, then we can't accept `alt` prop in **AirportIllustration**…

If one defines `alt` prop in **AirportIllustration**, it would still have `role="presentation"`, which is wrong. So we need to add `role` prop to **AirportIllustration** with `"presentation"` as default value, and remove the empty string for `alt` as default value.